css/style.css

```diff
@@ -0,0 +1,123 @@
+img {
+  display: block;
+}
+#navbar {
+  display: flex;
+  justify-content: flex-end;
+  color: white;
+  font-size: 15px;
+  background-color: #be3144;
+  height: 65px;
+  align-items: center;
+  position: fixed;
+  width: 100%;
+}
+body {
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+}
+ul {
+  list-style: none;
+  padding-right: 15px;
+}
+li a {
+  text-decoration: none;
+  color: white;
+}
+li {
+  display: inline;
+  padding: 10px;
+}
+#welcome-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(78, 77, 77);
+}
+.project-tile {
+  color: white;
+  font-size: 55px;
+  padding-bottom: 0px;
+}
+.web {
+  color: #be3144;
+  font-family: italic;
+  font-size: 40px;
+}
+#project-tile {
+  text-align: center;
+  background-color: #45567d;
+  /* width: 300px */
+}
+#projects {
+  text-align: center;
+  background-color: #45567d;
+  width: 100%;
+  display: inline;
+  justify-content: space-between;
+}
+.project_details {
+  display: inline;
+}
+.project-image {
+  height: calc(100%-6rem);
+  width: 25%;
+  object-fit: cover;
+}
+.bottom-text a {
+  font-size: 20px;
+  text-decoration: none;
+  color: white;
+  background-color: #45567d;
+}
+.bottom-text {
+  background-color: #45567d;
+  height: 60px;
+  align-content: center;
+  vertical-align: center;
+  padding-top: 30px;
+  width: 25%;
+}
+@media (max-width: 1500px) {
```

Comment:
The design
![designnn](https://user-images.githubusercontent.com/104437323/206456664-6c73df58-5f71-4965-96c2-e1f2a7490340.GIF)
- [ ] As stated the project requirement you must stick to the design as much as possible. You chooses a break point is not responsive.
Your work
![you design](https://user-images.githubusercontent.com/104437323/206460381-6103c63e-1e63-4ee6-b5db-11c7bfd1c756.GIF)